pxr/usd/pcp/layerStackRegistry.h

```diff
@@ -112,8 +112,8 @@ class Pcp_LayerStackRegistry : public TfRefBase, public TfWeakBase {
     PcpLayerStackPtr _Find(const PcpLayerStackIdentifier&) const;
 
     // Remove the layer stack with the given identifier from the registry.
-    void _Remove(const PcpLayerStackIdentifier&,
-                 const PcpLayerStack *);
+    void _SetLayersAndRemove(const PcpLayerStackIdentifier&,
+                             const PcpLayerStack *);
```

Comment:
Can we update the comment at line 135 to accommodate the renamed function?

Reply:
Thanks for catching this. Done.